agent/custom/auto_battle_repeat_action.py

```
@@ -63,6 +64,20 @@ class AutoBattleRepeatAction(CustomAction):
+
+        # "原生自动战斗次数" option 通过 attach 注入 battle_count（attach 与 action
+        # 是不同字段, 不会被 "Chaldea导入" 等 option 对 action 的 override 覆盖）。
+        # 优先级: custom_action_param > 节点 attach。
+        if "battle_count" not in param:
+            try:
+                node_data = context.get_node_data("原生自动战斗_多次入口") or {}
+                attach = node_data.get("attach") or {}
+                if "battle_count" in attach:
+                    param["battle_count"] = attach["battle_count"]
+                    mfaalog.info(f"[auto_battle_repeat] battle_count 来自节点 attach: {attach['battle_count']}")
+            except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)